internal/services/jsonrpc/server.go

```diff
@@ -56,6 +58,14 @@ type Server struct {
 	spec           json.RawMessage
 	authSHA        []byte
 	tlsCfg         *tls.Config
+
+	// UNSTABLE: this is not much more than a placeholder to ensure we can add
+	// our own metrics to the global prometheus metrics registry.
+	metrics map[string]Metrics
+}
+
+type Metrics interface {
+	Inc()
```

Comment:
Here's some PoC stuff.  I marked this counter-type metric as unstable as we should fully specify all of kwild's metrics and organize them better within a hierarchy of namespaces.